composer.json

```diff
@@ -25,45 +25,45 @@
         "ext-dom": "*",
         "ext-json": "*",
         "ext-sqlite3": "*",
-        "acquia/coding-standards": "^0.7.0",
-        "composer/composer": "~2.2.5",
+        "acquia/coding-standards": "^0.8.0",
+        "composer/composer": "~2.3.5",
         "cweagans/composer-patches": "^1.7",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
-        "ergebnis/composer-normalize": "^2.0.0",
-        "hassankhan/config": "^2.1",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+        "ergebnis/composer-normalize": "~2.15.0",
+        "hassankhan/config": "^3.0",
         "mglaman/drupal-check": "^1.3",
-        "myclabs/php-enum": "^1.7",
+        "myclabs/php-enum": "^1.8",
         "oscarotero/env": "^1.2",
-        "php-coveralls/php-coveralls": "^2.2",
-        "php-parallel-lint/php-console-highlighter": "^0.5.0",
-        "php-parallel-lint/php-parallel-lint": "^1.2.0",
-        "phploc/phploc": "7.0.2",
-        "phpmd/phpmd": "^2.6",
-        "symfony/config": "^4.1",
-        "symfony/console": "^4.1",
-        "symfony/dependency-injection": "^4.1",
+        "php-coveralls/php-coveralls": "^2.5",
+        "php-parallel-lint/php-console-highlighter": "^1.0",
+        "php-parallel-lint/php-parallel-lint": "^1.3",
+        "phploc/phploc": "^7.0",
+        "phpmd/phpmd": "^2.12",
+        "stecman/symfony-console-completion": "^0.11.0",
+        "symfony/config": "^5.4",
+        "symfony/console": "^5.4",
+        "symfony/dependency-injection": "^5.4",
         "symfony/expression-language": "^5.4",
-        "symfony/filesystem": "^4.1",
-        "symfony/finder": "^4.1",
-        "symfony/http-client": "^5.1",
-        "symfony/http-kernel": "^4.1",
-        "symfony/options-resolver": "^4.2",
-        "symfony/phpunit-bridge": "^4.2",
-        "symfony/process": "^4.1",
-        "symfony/yaml": "^4.1",
+        "symfony/filesystem": "^5.4",
+        "symfony/finder": "^5.4",
+        "symfony/http-client": "^5.4",
+        "symfony/options-resolver": "^5.4",
+        "symfony/phpunit-bridge": "^6.0",
+        "symfony/process": "^5.4",
+        "symfony/yaml": "^5.4",
         "webdriver-binary/binary-chromedriver": "^6.1.0",
-        "weitzman/drupal-test-traits": "^1.3",
+        "weitzman/drupal-test-traits": "^1.5",
         "zumba/amplitude-php": "^1.0"
     },
     "conflict": {
         "symfony/symfony": "*"
     },
     "require-dev": {
-        "brainmaestro/composer-git-hooks": "^2.5",
+        "brainmaestro/composer-git-hooks": "^2.8",
         "phan/phan": "^5.3",
         "phpspec/prophecy-phpunit": "^2.0",
-        "phpstan/phpstan": "^1.4",
-        "phpunit/phpunit": "^9"
+        "phpstan/phpstan": "^1.5",
```

Comment:
Please make sure there are no breaking changes in this version of PHPStan. In theory, a minor version update shouldn't have any, but if it did, it would cause a lot of headaches.